CSharp/BatchExplorer/Models/ModelBase.cs

```diff
@@ -106,13 +106,17 @@ private List<PropertyModel> ObjectToPropertyModelRecursive(object o, List<string
                         //Don't track null properties in order to keep the list short
                         //propertyModel = new SimplePropertyModel(propInfo.Name, string.Empty);
                     }
+                    else if ((propertyType == typeof(TimeSpan) || propertyType == typeof(TimeSpan?)) && propertyValue.Equals(TimeSpan.MaxValue))
+                    {
+                        propertyModel = new SimplePropertyModel(propInfo.Name, "Unlimited");
```

Comment:
@paselem I think unlimited is okay as well... if you're okay with it I will merge this.  Let me know